docs/api/combineManagers.md

```diff
@@ -0,0 +1,26 @@
+# `combineManagers(...managers)`
```

Comment:
@matthewwithanm IMO that would be particularly confusing and potentially confusing, make people think they naturally run in workers for performance or something. But I fee ya!